sycl/include/CL/sycl/detail/kernel_program_cache.hpp

```diff
@@ -33,6 +38,7 @@ class KernelProgramCache {
   struct EntityWithState {
     std::atomic<T *> Ptr;
     std::atomic<int> State;
+    std::unique_ptr<BuildResultT> BuildResult;
```

Comment:
`EntityWithState` name is a bit unclear for a struct that has `BuildResult` field.

Reply:
Renamed.